src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,25 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.Arrays;
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    static final String NATIONALITY = "Ukrainian";
+    static final int LOW_AGE = 35;
+    static final int LIVE_IN_COUNTRY = 10;
```

Comment:
dont forget about access modifiers 
```suggestion
    private static final String NATIONALITY = "Ukrainian";
    private static final int LOW_AGE = 35;
    private static final int LIVE_IN_COUNTRY = 10;
```